src/ExampleApp.cpp

```diff
@@ -117,143 +132,316 @@ ExampleApp::ExampleApp(Eegeo::EegeoWorld* pWorld,
     const float interestPointAltitudeMeters = 2.7f;
     const float cameraControllerOrientationDegrees = 0.0f;
     const float cameraControllerDistanceFromInterestPointMeters = 1781.0f;
-    
+
     m_pCameraControllerFactory = new Examples::DefaultCameraControllerFactory(
-                                                                    terrainModelModule,
-                                                                    mapModule,
-                                                                    *m_pCameraTouchController,
-                                                                    m_screenPropertiesProvider,
-                                                                    globeCameraControllerConfig,
-                                                                    twoFingerPanTiltEnabled,
-                                                                    interestPointLatitudeDegrees,
-                                                                    interestPointLongitudeDegrees,
-                                                                    interestPointAltitudeMeters,
-                                                                    cameraControllerOrientationDegrees,
-                                                                    cameraControllerDistanceFromInterestPointMeters);
+    terrainModelModule,
+    mapModule,
+    *m_pCameraTouchController,
+    m_screenPropertiesProvider,
+    globeCameraControllerConfig,
+    twoFingerPanTiltEnabled,
+    interestPointLatitudeDegrees,
+    interestPointLongitudeDegrees,
+    interestPointAltitudeMeters,
+    cameraControllerOrientationDegrees,
+    cameraControllerDistanceFromInterestPointMeters);
     
     m_pLoadingScreen = CreateLoadingScreen(screenProperties, eegeoWorld.GetRenderingModule(), eegeoWorld.GetPlatformAbstractionModule());
     
     m_pExampleController = new Examples::ExampleController(*m_pWorld,
-                                                           view,
-                                                           *m_pCameraControllerFactory,
-                                                           *m_pCameraTouchController);
-
-	//register all generic examples
+    view,
+    *m_pCameraControllerFactory,
+    *m_pCameraTouchController);
+
+
+	#ifdef CARDBOARD
+	m_pStreamingVolume = Eegeo_NEW(Eegeo::Streaming::CameraFrustumStreamingVolume)(mapModule.GetResourceCeilingProvider(),
```

Comment:
There's a lot of VR specific concerns in here and consequently lots of #ifdefs to conditionally remove it. Can we move some of it out to other classes. Modules and/or Controllers?